### DISCUSSION TITLE
How to set text color based on current theme?

### DISCUSSION THREAD

**Discussion by aneeskA** [fyne-io/fyne#4940] [General] [answered]
I am using `NewText` to create a text with custom font size. I want to set the color based on the current theme selected. How do I do this?

Current code
```go
count := canvas.NewText("?", color.White)
```

What I want to do
```go
count := canvas.NewText("?", app.Theme().ForegroundColor())
```

Note: I am not overriding the OS theme. I use the System theme as is.

**Comment by andydotxyz** [comment, marked as answer]
Just use `theme.ForegroundColor()`
